docs/DOCKER.md

```diff
@@ -69,11 +69,17 @@ docker build -t <TAG_OF_THE_IMAGE> .
 For example, if you've created a custom version of MongoDB connector, you can tag it with the following command:
 
 ```
-docker build -t connector/custom-mongodb:1.0 -f .
+docker build -t connector/custom-mongodb:1.0 .
 ```
 
 You can later use `<TAG_OF_THE_IMAGE>` instead of `docker.elastic.co/enterprise-search/elastic-connectors:<VERSION>-SNAPSHOT` in the next step to run the Docker image.
 
+If you're an Elastic employee, you may want to build a Chainguard-based image using `Dockerfile.wolfi`:
```

Comment:
I'm thinking we'll want to get feedback from @elastic/ingestion-team on whether we want to call this out here (or not), and if we do, whether we need to make any changes to the wording.

/cc @seanstory

Reply:
Just wonder if there is still a way to build an image using the old Dockerfile? If yes, should we leave it? Or using chaingguard is the only option we have?

Reply:
yep, the old Dockerfile is being left in place in this PR; my concern here was more to get someone from your team to confirm that we want to call out the internal-only image in this README in this way

Reply:
@vidok Yep, it's still possible to build an image using the current `Dockerfile` (so I preserved the existing documentation, some lines above).

Please also note that the same can be achieved with `make docker-build` (no changes here), and it's now also possible to build a Wolfi image with `DOCKERFILE_PATH=Dockerfile.wolfi make docker-build`: I haven't documented it because those `docker-*` Make targets are not documented at the moment (and it's OK in my opinion).